webui/react/src/pages/ModelDetails/ModelHeader.module.scss

```diff
@@ -1,25 +1,3 @@
 .base {
-  background-color: var(--theme-colors-monochrome-17);
-  border-bottom: 1px solid var(--theme-colors-monochrome-12);
-  width: 100%;
-
-  .headerContent {
-    margin: 0 12px;
-    padding: 12px;
-
-    .mainRow {
-      align-items: center;
-      display: flex;
-      justify-content: space-between;
-
-      .nameAndIcon {
-        margin-block: 12px;
-
-        .name {
-          font-size: 24px;
-          margin: 0;
-        }
-      }
-    }
-  }
+  padding: var(--spacing-xl-3);
```

Comment:
Considered moving the padding out to other determined or hew components, but ultimately I think it makes sense here since it is styling that's specific to this header (some other Page header components are flush without padding).